### DISCUSSION TITLE
Unable to use this due to an error.

### DISCUSSION THREAD

**Discussion by CallumAustinTaylor** [Extraltodeus/multi-subject-render#18] [Q&A] [answered]
Hey, I figure I might post this here since I can't find it anywhere else; anywho, it will give the error on the GUI under the image output of txt2img "AttributeError: 'StableDiffusionProcessingTxt2Img' object has no attribute 'sampler_name'". I'm frankly not the best with computers in this regard to solving these sorts of errors, and I figure I might ask for help here. I would much appreciate some help, and I'll post the specs of my CPU, GPU, RAM and OS below as well as some additional info it spat out in the webui-user.bat file.

Nvidia GTX 1060 6Gb 
AMD Ryzen 2700x 
Windows 10. 

Error completing request
Arguments: ('', '', 'None', 'None', 20, 0, False, False, 1, 1, 7, -1.0, -1.0, 0, 0, 0, False, 512, 512, False, 0.7, 0, 0, 4, False, False, False, '', 1, '', 0, '', True, False, False, '', 5, 24, 12.5, 1000, 'DDIM', 0, 64, 64, '', 64, 7.5, 0.42, 'DDIM', 64, 64, 1, 0, 92, True, True, True, False, False, '', 5, 24, 12.5, 1000, 'DDIM', 0, 64, 64, '', 64, 7.5, 0.42, 'DDIM', 64, 64, 1, 0, 92, True, True, True, False, False) {}
Traceback (most recent call last):
  File "C:\Users\xyenf\Desktop\stable-diffusion-webui\modules\ui.py", line 185, in f
    res = list(func(*args, **kwargs))
  File "C:\Users\xyenf\Desktop\stable-diffusion-webui\webui.py", line 54, in f
    res = func(*args, **kwargs)
  File "C:\Users\xyenf\Desktop\stable-diffusion-webui\modules\txt2img.py", line 45, in txt2img
    processed = modules.scripts.scripts_txt2img.run(p, *args)
  File "C:\Users\xyenf\Desktop\stable-diffusion-webui\modules\scripts.py", line 290, in run
    processed = script.run(p, *script_args)
  File "C:\Users\xyenf\Desktop\stable-diffusion-webui\extensions\multi-subject-render\scripts\multirender.py", line 170, in run
    o_sampler_name = p.sampler_name
AttributeError: 'StableDiffusionProcessingTxt2Img' object has no attribute 'sampler_name'

Thanks for reading!

**Comment by Extraltodeus** [comment, marked as answer]
Try to update your automatic1111 repository. An update has been made that create that bug.

**Comment by amiketanner** [comment]
Can anyone help with what needs to be changed or rolled back to get this to work when added as extension in auto's webui?

**Reply by Extraltodeus** [reply]
Try to :
- update your webui with a "git pull" from the commandline within your webui folder.
- delete the extension folder (so this extension, not the whole extension folder)
- reinstall it like you would normally

**Comment by amiketanner** [comment]
Thank you, I ended up creating a new, clean automatic1111 instance and it works. I was thinking that there may have been an update from automatic1111 that created an incompatibility, but it must have been something else that I had done.
